### ISSUE TITLE
Ability to select FileSets to be displayed in UV

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#3457]
### Descriptive summary
Soon, there will be the possibility of multiple of the same file to be attached to the same work, one accessibility remediated file and one original file. The accessibility review team would like a mechanism to be able to select specific FileSets on a work to be displayed in UV. If no FileSets are manually selected, the earliest uploaded FileSet should be used. Unless multiple FileSets are selected to be displayed only one FileSet (the original) should be displayed.

### Expected behavior
UV follows these scenarios:
- No FileSet is selected as display: Display only the oldest uploaded FileSet
- Any number of FileSets are selected for display: Display only those FileSets

### Related work
#3454 - Required
